file-formats/gsmg/examples/z_aff_example_gsmg.gsmg.json

```diff
@@ -0,0 +1,12 @@
+{
+  "formatVersion": "1",
+  "header": {
+    "description": "For testing",
+    "originalLanguage": "en",
+    "abapLanguageVersion": "standard"
+  },
+  "generalInformation": {
+    "type": "USAGE",
+    "detailedDescription": "This Metric Group is for testing"

```

Comment:
What is this description about? Is it an extension of the description field in the header? So like the name says, is it for describing the object in more detail than the header-description allows?

Reply:
Exactly right :-)

Reply:
Just some additional questions: Did you thought about extending the standard description to 255 characters. As far as I know, this should be possible.

If it is more a documentation. Did you consider to provide documentation using KTD?